pkg/hostagent/mount.go

```diff
@@ -48,7 +48,7 @@ func (a *HostAgent) setupMount(ctx context.Context, m limayaml.Mount) (*mount, e
 		RemotePath: expanded,
 		Readonly:   !m.Writable,
 		// NOTE: allow_root requires "user_allow_other" in /etc/fuse.conf
-		SSHFSAdditionalArgs: []string{"-o", "allow_root"},
+		SSHFSAdditionalArgs: []string{"-o", "allow_other"},
```

Comment:
We should update the doc to clarify that "The mount point might be accessible from all UIDs in the guest".

https://github.com/lima-vm/lima/blob/72aea5c5f13687bd4e400a45e7f24abb0476468d/pkg/limayaml/default.yaml#L38

The behavior may change again when we migrate to Samba (#188), though.